batch_import/EXAMPLE.md

```diff
@@ -0,0 +1,210 @@
+Batch Import Concrete Example
+========================
+
+### What's Needed
+
+1. A registered Camio account
+2. A Camio Box (either [physical](https://www.camio.com/box) or a [VM](https://www.camio.com/box/vm))
+3. An OAuth token for your Camio account (gotten from [the integrations page](https://www.camio.com/settings/integrations))
+4. The Camio Box IP-address (explained in the [section](#get-the-camio-box-ip-address))
+5. A directory of videos that you wish to process through Camio
+6. A regular-expression describing how to parse your input videos ([described here](#constructing-the-file-parsing-regex))
+7. Python Version 2.7 (installed by default on OSX and Linux, can be obtained from the [python website](https://www.python.org/downloads/windows/) for Windows)
+
+Once you have all of the items listed above, you can start to batch-import videos to Camio
+
+### Instructions
+
+#### Boot-Up Your Camio Box
+
+Your Camio Box needs to be powered on, registered under your account, and on the same local-network as the computer running the batch-importer script.
+Your Camio Box can be registered through [the /box/register page](https://www.camio.com/box/register).
+
+#### Obtain the video-import Script and the Camio-specific Hooks Code
+
+Start by cloning the [Camio examples](https://www.github.com/CamioCam/examples) repository, go into the 
+[examples/batch_import/video-importer](examples/batch_import/video-importer) directory.
+
+```sh
+$ git clone https://www.github.com/CamioCam/examples
+Cloning into 'examples'...
+remote: Counting objects: 394, done.
+remote: Compressing objects: 100% (127/127), done.
+remote: Total 394 (delta 68), reused 0 (delta 0), pack-reused 267
+Receiving objects: 100% (394/394), 87.31 KiB | 138.00 KiB/s, done.
+Resolving deltas: 100% (218/218), done.
+Checking connectivity... done.
+$ cd examples/batch_import/video-importer
+$ pwd
+/home/user/examples/batch_import/video-importer
+$ ls -l
+total 48
+-rw-r--r--  1 user  staff  1062 May  1 15:32 LICENSE
+-rw-r--r--  1 user  staff  7145 May  1 18:57 README.md
+-rw-r--r--  1 user  staff  9408 May  1 18:57 importer.py
+
+```
+
+#### Set the Necessary Environment Variables
+
+For the video-import script to work properly with Camio and the [Camio Hooks](examples/batch_import/camio_hooks.py) module, a few environment 
```

Comment:
camio_hooks.py (the python module so as not to confuse with the general concept of Camio Hooks)